subsys/nv_config/src/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+zephyr_include_directories(${ZEPHYR_BASE}/ext/utils)
+
+#mbedtls-base64 library header
+zephyr_include_directories($ENV{ZEPHYR_BASE}/ext/lib/crypto/mbedtls/include)
```

Comment:
why do we need mbedtls here? none of the conf_* headers file are using this. Can we make this implementation specific?

Reply:
In config.c mbdetls base64 library is called.